src/include/compat.h

```diff
@@ -259,10 +262,29 @@ struct iovec {
   size_t iov_len;
 };
 
+#define S_IFLNK   0120000
```

Comment:
This was defined in Inode.h -- do we need to do it here again? It would be best to arrange things so that it's only defined once.

Reply:
Thanks for bringing this up, I'll avoid the redundant definitions.